### DISCUSSION TITLE
I'm getting error in Extract the content for all components , but in catalogue  it extracting the content without error

### DISCUSSION THREAD

**Discussion by Kurra432** [learndevopsonline/learndevopsonline#1958] [Q&A] [answered]
This is from the user component it's not extracting 

![image](https://github.com/user-attachments/assets/1d10315f-f952-4ce3-8237-b13b12d91c8f)

This from the catalogue component it's extracting

![image](https://github.com/user-attachments/assets/e525055d-3545-4028-9283-fd7c37f61742)

![image](https://github.com/user-attachments/assets/ea745522-3e0f-489d-b27c-51b4c21c20c3)

**Comment by chaithanya1812** [comment, marked as answer]
Hello @Kurra432 ,
## for catalogue it won't works, But In your case it work because the zip-file  of catalogue is already is present in server.
![image](https://github.com/user-attachments/assets/9fb9989a-fd40-46ea-a908-375ac427ca59)
## I removed the catalogue.zip file then it won't works!!
![image](https://github.com/user-attachments/assets/e3d2d3eb-c541-404b-a825-70e7e6685289).

## same case for the user component.
# Issue.
![image](https://github.com/user-attachments/assets/c0b072bb-1b2f-4255-bc71-cc257b894150)
